2022-08-24-Pods-BlogPost/sample_ml-model/run.sh

```diff
@@ -1,21 +1,31 @@
 #!/bin/bash
 
 zip lambda.zip train.py
+zip infer.zip infer.py
 
 # push training data to S3
 awslocal s3 mb s3://pods-test
 awslocal s3 cp lambda.zip s3://pods-test/lambda.zip
+awslocal s3 cp infer.zip s3://pods-test/infer.zip
 awslocal s3 cp digits.rst s3://pods-test/digits.rst
 awslocal s3 cp digits.csv.gz s3://pods-test/digits.csv.gz
 
 # define lamba function to training the ML data
 awslocal lambda create-function --function-name func1 \
```

Comment:
tiny nit: Maybe we could name the Lambdas as `ml-train`/`ml-predict` instead of `func1`/`func2`, to make it a bit more clear what the scope of these functions is.